github/ipfs.yml

```diff
@@ -340,9 +340,6 @@ repositories:
     advanced_security: false
```

Comment:
@galargh feel free to drop the entire `2022.ipfs.camp` key, it was an empty repo that I've since removed. ty

Reply:
Noted, thanks. I'll clean up teams I was told are no longer relevant after this one's merged.